GVRf/Framework/src/org/gearvrf/GVRRenderData.java

```diff
@@ -323,6 +323,31 @@ public boolean getAlphaBlend() {
     public void setAlphaBlend(boolean alphaBlend) {
         NativeRenderData.setAlphaBlend(getPtr(), alphaBlend);
     }
+
+    /**
+     * @return The OpenGL draw mode (e.g. GL_TRIANGLES).
+     */
+    public int getDrawMode() {
+        return NativeRenderData.getDrawMode(getPtr());
+    }
+
+    /**
+     * Set the draw mode for this mesh. Default is GL_TRIANGLES.
+     * 
+     * @param drawMode
+     */
+    public void setDrawMode(int drawMode) {
+        if (drawMode != GL_POINTS && drawMode != GL_LINES
+                && drawMode != GL_LINE_STRIP && drawMode != GL_LINE_LOOP
+                && drawMode != GL_TRIANGLES && drawMode != GL_TRIANGLE_STRIP
+                && drawMode != GL_TRIANGLE_FAN) {
+            throw new IllegalArgumentException(
+                    "drawMode must be one of GL_POINTS, GL_LINES, GL_LINE_STRIP, GL_LINE_LOOP, GL_TRIANGLES, GL_TRIANGLE_FAN, GL_TRIANGLE_STRIP.  Value passed was: "
+                            + drawMode);
```

Comment:
Good point on the decimal value of drawMode.  So it sounds like just truncating the message to say that its not one of the GL_* values would be sufficient.